Index: src/test/java/ru/skypro/homework/HomeworkApplicationTests.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.skypro.homework;\r\n\r\nimport org.junit.jupiter.api.Test;\r\nimport org.springframework.boot.test.context.SpringBootTest;\r\n\r\n@SpringBootTest\r\nclass HomeworkApplicationTests {\r\n\r\n    @Test\r\n    void contextLoads() {\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/ru/skypro/homework/HomeworkApplicationTests.java b/src/test/java/ru/skypro/homework/HomeworkApplicationTests.java
--- a/src/test/java/ru/skypro/homework/HomeworkApplicationTests.java	(revision b9b8d64e1be130887395efc50b8371adfdaace3b)
+++ b/src/test/java/ru/skypro/homework/HomeworkApplicationTests.java	(date 1691397120508)
@@ -1,13 +1,0 @@
-package ru.skypro.homework;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class HomeworkApplicationTests {
-
-    @Test
-    void contextLoads() {
-    }
-
-}
